.github/workflows/NotificationSystem.yaml

```
@@ -84,10 +84,12 @@ jobs:
       run: |
         Get-Command func
         func --version
-    - name: Test
-      working-directory: NotificationSystem/NotificationSystemTests
-      # TODO: Make the Cosmos test run in a github hosted runner
+    - name: Unit Tests
+      working-directory: NotificationSystem/NotificationSystem.Api.Tests.Unit
       run: dotnet test --filter "Category!=Cosmos"
```

```suggestion
      run: dotnet test
```
There are no Cosmos tests in NotificationSystem/NotificationSystem.Api.Tests.Unit after the updates in this PR.